package.json

```diff
@@ -17,7 +17,9 @@
     "eslint": "^5.11.0",
     "eslint-plugin-cypress": "^2.2.1",
     "http-server": "^0.11.1",
+    "jszip": "^3.5.0",
```

Comment:
Why not node's built-in zlib?
1. nodes zlib is hard to use and not documented at all. It confuses me and I failed to get it work after several tries
2. zlib doesnot support `.zip` format, it only supports `.gz` and others. What we were doing was `.zip` files. (While it remains unclear whether google's extension store accepts other formats, staying on `zip` is always a safe bet)